index.html

```diff
@@ -5224,6 +5224,79 @@ <h2>IANA Considerations</h2>
         </li>
       </ul>
 
+<section id="media-type">
+  <h2>The <code>application/credential+json</code> Media Type</h2>
+  <p>
+    This specification registers the <code>application/credential+json</code> MIME Media Type specifically for
+    identifying documents
+    conforming to the Verifiable Credentials format.
+  </p>
+  <table>
+    <tr>
+      <td>Type name: </td>
+      <td>application</td>
+    </tr>
+    <tr>
+      <td>Subtype name: </td>
+      <td>credential+json</td>
+    </tr>
+    <tr>
+      <td>Required parameters: </td>
+      <td>None</td>
+    </tr>
+    <tr>
+      <td>Optional parameters: </td>
+      <td>
+        profile: The profile parameter for the application/credential+json
+        media type allows one or more profile URIs to be specified. These
+        profile URIs have the identifier semantics defined in [[!RFC6906]].
+        The "profile" media type parameter MUST be quoted. It contains a
+        non-empty list of space-separated URIs (the profile URIs).
+        <pre>profile-param = "profile=" profile-value
+profile-value = <"> profile-URI 0*( 1*SP profile-URI ) <">
+profile-URI   = URI</"></"></pre>
+        The "URI" in the above grammar refers to the "URI" as defined in
+        Section 3 of [[!RFC3986]].
+      </td>
+    </tr>
+    <tr>
+      <td>Encoding considerations: </td>
+      <td>
+        Resources that use the "<code>application/credential+json</code>"
```

Comment:
```suggestion
        Resources that use the "<code>application/vc+json</code>"
```

Reply:
I think this suggestion further conflates `credential` and `verifiable credential`... I would prefer to be able to uniquely identify "content of the data model" from "content of the data model with protection".

Reply:
However, that ship may have sailed in which case perhaps we should adopt your suggested change.

Reply:
I agree that we should not conflate the media types for a credential (the data structure to be secured) and a verifable credential (a secured credential).  "credential" is correct for the former and "verifiable-credential" or "vc" is correct for the latter.

Reply:
See comment here: https://github.com/w3c/vc-data-model/pull/1000#discussion_r1065927379

Reply:
I would also "+1" for `application/credential` being the data model, and `application/vc` being a secured credential.

Reply:
Ok, so this PR is just about `application/credential+json` which is a JSON-LD compatible object. Withdrawing my suggestion for doing `application/vc` in this PR.